src/playbooks/deploy/metadata.obsah.yaml

```diff
@@ -11,6 +11,19 @@ variables:
     help: Number of workers for Puma.
   pulp_worker_count:
     help: Number of Pulp workers. Defaults to 8 or the number of CPU cores, whichever is smaller.
+  external_authentication:
+    help: External authentication method to use
+    choices:
+      - ipa
+      - ipa_with_api
```

Comment:
What is the difference between the two? `ipa` is UI only, while `ipa_with_api` does UI and API?